ui/src/common/layouts/CollectionsMenu/CollectionsMenu.jsx

```diff
@@ -68,12 +70,35 @@ function CollectionsMenu({ currentPathname }) {
         </CollectionLink>
       </Col>
       <Col>
-        <CollectionLink
-          active={activeCollection === INSTITUTIONS_PID_TYPE}
-          to={INSTITUTIONS}
+        <Menu
+          className="__CollectionsDropdownMenu__ mh4 m-mh2"
+          theme="dark"
+          mode="horizontal"
+          selectable={false}
+          inlineIndent={0}
         >
-          Institutions
-        </CollectionLink>
+          <Menu.SubMenu key="More" title="More...">
+            <Menu.Item key="more.institutions">
+              <CollectionLink
+                active={activeCollection === INSTITUTIONS_PID_TYPE}
+                to={INSTITUTIONS}
+                isDropdownLink
+              >
+                Institutions
+              </CollectionLink>
+            </Menu.Item>
+            <Menu.Item key="more.experiments">
+              <CollectionLink
+                active={activeCollection === EXPERIMENTS_PID_TYPE}
+                to={EXPERIMENTS}
+                isDropdownLink
+                newCollection
```

Comment:
And I would put new tag for `More...` since it's not visible enough inside the dropdown.